pylib/anki/tags.py

```diff
@@ -103,11 +87,11 @@ def bulk_update(
     def rename_tag(self, old: str, new: str) -> int:
         "Rename provided tag, returning number of changed notes."
         escaped_name = re.sub(r"[*_\\]", r"\\\g<0>", old)
-        escaped_name = '"{}"'.format(escaped_name.replace('"', '\\"'))
-        nids = self.col.find_notes("tag:" + escaped_name)
+        quote_escaped = escaped_name.replace('"', '\\"')
+        nids = self.col.find_notes(f'tag:"{quote_escaped}"')
```

Comment:
Thanks to Rumo's recent change, we can now skip the manual escaping:

```
search = self.col.backend.filter_to_search(FilterToSearchIn(tag=old))
nids = self.col.find_notes(search)
```

Reply:
Fantastic!
Though we also need to escape the name manually for bulk_update.

Reply:
Ah, that's a shame. Maybe we can change bulk_update to an enum in the future and allow passing in a literal text. Not something we need to do for this PR though.